lib/beaker-answers/answers.rb

```diff
@@ -4,12 +4,13 @@ module BeakerAnswers
   class Answers
 
     DEFAULT_ANSWERS =  StringifyHash.new.merge({
+      :q_install                                     => 'y',
       :q_puppet_enterpriseconsole_auth_user_email    => 'admin@example.com',
       :q_puppet_enterpriseconsole_auth_password      => '~!@#$%^*-/ aZ',
       :q_puppet_enterpriseconsole_smtp_port          => 25,
       :q_puppet_enterpriseconsole_smtp_use_tls       => 'n',
       :q_verify_packages                             => 'y',
-      :q_puppetdb_password                           => '~!@#$%^*-/ aZ',
+      :q_puppetdb_database_password                  => "'~!@#$%^*-/ aZ'",
```

Comment:
@tvpartytonight I'm still missing something here about the double quoting in this change.  None of the other database passwords are double quoted?